lib/device/network.js

```diff
@@ -4,244 +4,224 @@ var util = require('util');
 var EventEmitter = require('events').EventEmitter;
 var exec = require('child_process').exec;
 var _ = require('lodash');
-var network = require('network');
+var networkModule = require('network');
```

Comment:
Yeah, removed the calls to third party services in the mean time, implementing that function on the network package means implementing several functions underneath, think we could copy/implement all those in **piwifi**